apps/web/src/app/tournament/[id]/create/_components/TournamentCreateClient.tsx

```
@@ -96,23 +96,26 @@ function TournamentCreateClient({ tournamentId }: TournamentCreateClientProps) {
             participants={participants}
             inviteCode={pending?.inviteCode ?? ''}
             inviteExpiresAt={pending?.inviteExpiresAt ?? ''}
+            depositDeadline={!ownerStarted && !isDepositClosed ? depositDeadline : undefined}
```

ESLint

[error] Unexpected use of undefined.

(no-undefined)